README.md

```diff
@@ -1,14 +1,14 @@
-# krane
+# kubernetes-deploy
 
-As this project approaches the v1.0 milestone, we're excited to announce that `kubernetes-deploy` will be [officially renamed as `krane`](https://github.com/Shopify/kubernetes-deploy/issues/30#issuecomment-468750341). Follow the [1.0 requirement label](https://github.com/Shopify/kubernetes-deploy/issues?q=is%3Aissue+is%3Aopen+label%3A%22%3Arocket%3A+1.0+requirement%22) to keep up with the progress.
+As this project approaches the v1.0 milestone, we're excited to announce that `krane` will be [officially renamed as `krane`](https://github.com/Shopify/kubernetes-deploy/issues/30#issuecomment-468750341). Follow the [1.0 requirement label](https://github.com/Shopify/kubernetes-deploy/issues?q=is%3Aissue+is%3Aopen+label%3A%22%3Arocket%3A+1.0+requirement%22) to keep up with the progress.
```

Comment:
needs to be re-worded

Reply:
I changed the sentence for this phrase:

```
This was the previous name of this project! If you're curious about the rename to `Krane`, check [this issue](https://github.com/Shopify/kubernetes-deploy/issues/30#issuecomment-468750341) and [this issue](https://github.com/Shopify/kubernetes-deploy/issues?q=is%3Aissue+is%3Aopen+label%3A%22%3Arocket%3A+1.0+requirement%22) for context.
```

Could I improve it somehow?